src/monet.js

```diff
@@ -99,10 +158,10 @@
     var foldLeft = function (fn, acc, l) {
         function fL(acc, l) {
             return l.isNil ?
-                Return(acc) :
-                Suspend(function () {
-                    return fL(fn(acc, l.head()), l.tail())
-                })
+              Return(acc) :
+              Suspend(function () {
+                  return fL(fn(acc, l.head()), l.tail())
+              })
```

Comment:
I think it's high time we decided on a fixed indentation. It seems like it's 4 spaces in general, but you're using 2 spaces here, around line 173 and 188